tekton/resources/nightly-release/tasks.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: golang-build
```

Comment:
not sure why we want to duplicate this here vs applying the version in the catalog?

Reply:
I should have left a comment on this :pray:
The version in the catalogue is different, this one uses pipeline resources, and the only place it exists today is in the cluster :(
I do want to remove all pipeline resources from the release process, but that's a good sized chunk of work by itself, so I'll be working on that in another series of PRs.

Reply:
Yep discussed with @afrittoli, the goal is to not have those anymore *but* for now, we should make sure all that is on the cluster is stored here. And then we can migrate to depend on the catalog version.